jujugui/static/gui/src/app/views/topology/relation.js

```diff
@@ -837,7 +837,8 @@ YUI.add('juju-topology-relation', function(Y) {
 
       // Transform endpoints into a list of relatable services (to the
       // service).
-      var possible_relations = Y.Array.flatten(Y.Object.values(endpoints)).map(
+      var possible_relations = Y.Array.flatten(
```

Comment:
Ah, quite possibly, I was trying to do the object methods before I looked at the array stuff.